internal/dockercompose/client.go

```diff
@@ -285,6 +320,24 @@ func (c *cmdDCClient) dcOutput(ctx context.Context, configPaths []string, args .
 	return strings.TrimSpace(string(output)), err
 }
 
+// ParseComposeVersionOutput parses the raw output of `docker-compose --version` (v1) or `docker-compose version` (v2)
```

Comment:
Oops, yeah - will update.

For context: v2 _only_ supports `version` subcommand, while v1 can do both `version` subcommand (from 1.4.0+) and `--version` flag (from 1.0.0+). I originally had it try one first and then fall back if it failed, but then decided that was overkill since the `version` subcommand covers everything from 2015 onwards 😝

Reply:
Updated in ab2c5b662